src/Bookings.js

```diff
@@ -1,20 +1,39 @@
-import React from "react";
-import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import React, { useState, useEffect } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      .then(res => res.json())
+      .then(data => {
+        console.log(data);
+        setBookings(data);
+      });
+  }, []);
+
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    const selectedBooking = bookings.filter(
+      item =>
+        item.firstName.toLowerCase().includes(searchVal.toLowerCase()) ||
+        item.surname.toLowerCase().includes(searchVal.toLowerCase())
+    );
```

Comment:
This is really good! It's cool that you've lowercased _both_ the search terms and the fields you're searching over.

I also like that you're using `includes` instead of `===`, to allow users to search for any part of the string.

There could be a good argument for using `startsWith` as an alternative to `includes`, because this might reduce unwanted results in the search output. But this is the sort of thing you'd usually discuss with the customer, or your business analyst.